src/Game/ObjectFactory.cpp

```diff
@@ -308,18 +308,7 @@ GameObject* GameObjectFactory::createObject(unsigned int PID)
     }
     object->setPID(PID);
     object->setFID(proto->FID());
-    object->setCanWalkThru(proto->flags() & 0x00000010);
-    object->setCanLightThru(proto->flags() & 0x20000000);
-    object->setCanShootThru(proto->flags() & 0x80000000);
-
-    object->setTrans(GameObject::TRANS_DEFAULT);
-    if (proto->flags() & 0x00004000) object->setTrans(GameObject::TRANS_RED);
```

Comment:
He called parseFlags() below, which does the same as the replaced code, as far as I can see.